src/config/authConfig.ts

```diff
@@ -32,7 +32,13 @@ export const loginRequest: RedirectRequest = {
   state: LOGIN_STATE,
 };
 
-const apiScope = `api://${apiClientId}/.default`;
+let apiScope;
+
+if (apiClientId) {
+  apiScope = `api://${apiClientId}/.default`;
+} else {
+  apiScope = 'api://mad-learning/Read'
+}
```

Comment:
You can use a ternary instead to avoid having to declare the variable as mutable (`let`):
```suggestion
const apiScope = apiClientId
  ? `api://${apiClientId}/.default`
  ? 'api://mad-learning/Read;
```

Reply:
The second questionmark is supposed to be a ":" right? like this:

```suggestion
const apiScope = apiClientId
  ? `api://${apiClientId}/.default`
  : 'api://mad-learning/Read';
```